src/de/mod.rs

```diff
@@ -296,15 +296,6 @@ where
     T::deserialize(&mut de)
 }
 
-/// Deserialize an instance of type `T` from bytes of XML text.
-pub fn from_slice<'de, T>(s: &'de [u8]) -> Result<T, DeError>
-where
-    T: Deserialize<'de>,
-{
-    let mut de = Deserializer::from_slice(s);
-    T::deserialize(&mut de)
-}
-
```

Comment:
Still, the constructors were probably deleted in vain. What do they interfere with? It's still may be valuable to pass a byte array and allow quick_xml to detect encoding and parse it. If encoding is UTF-8, then borrowing is possible. @dralley, what do you think?

If it remains removed, then it is needed to fix dangling documentation links

Reply:
re: documention, whoops, yes, I can fix that

re: constructors, IMO it is unnecessary, this is why we have the decoding functions as separate now.  It's only an extra line or two of code and if we made a constructor which included that line it would need to be falliable and return Result, unlike the others.

Reply:
We should make the CI fail on documentation issues if possible ^^